src/ml_board/frontend/Dashboard/src/app/App.tsx

```diff
@@ -38,30 +36,30 @@ async function getUrlParamsOrLocalStorageData(searchParams: URLSearchParams, set
     let restApiUrl = searchParams.get("rest_endpoint")
 
     let settingConfigsInStorage = localStorage.getItem('SettingConfigs');
-    if(settingConfigsInStorage) {
+    if (settingConfigsInStorage) {
         settingConfigs = await JSON.parse(settingConfigsInStorage);
     }
 
     // in the else parts below: for now, I kept default values if the server is localhost or 127.0.0.1 - so we have ease in development. Let me know if it needs to be changed.
-    if(gridSearchId !== null) {
+    if (gridSearchId !== null) {
         settingConfigs.gridSearchId = gridSearchId;
     }
-    else if(!settingConfigsInStorage && (window.location.hostname === "localhost" || window.location.hostname === "127.0.0.1")) {
+    else if (!settingConfigsInStorage && (window.location.hostname === "localhost" || window.location.hostname === "127.0.0.1")) {
         settingConfigs.gridSearchId = "mlgym_event_subscribers";
     }
-    
-    if(socketConnectionUrl !== null) {
+
+    if (socketConnectionUrl !== null) {
         settingConfigs.socketConnectionUrl = socketConnectionUrl;
     }
-    else if(!settingConfigsInStorage && (window.location.hostname === "localhost" || window.location.hostname === "127.0.0.1")) {
-        settingConfigs.socketConnectionUrl = "http://"+window.location.hostname+":7000/";
+    else if (!settingConfigsInStorage && (window.location.hostname === "localhost" || window.location.hostname === "127.0.0.1")) {
+        settingConfigs.socketConnectionUrl = "http://" + window.location.hostname + ":7000/";
```

Comment:
we should never connect to port 7000 by default. If no websocket endpoint is specified we should instead show the "settings pop-up" and force the user to specify an endpoint.

Reply:
@OsamaMSoliman As we discussed, I have resolved this one. We just needed to delete `else if` sections

Reply:
Perfect, thanks!
I will update and do one final push then ask max to review again!